Liquibase native compilation broken since 1.11.1.Final: NullPointerException in LiquibaseProcessor#nativeImageConfiguration

**Describe the bug**
Since the version 1.11.1.Final I cannot compile native with liquibase. It was working fine in 1.11.0.Final. I suspect the issue started with #14500 (because of the stacktrace). This only happen if I have a `<createView` because if I comment it out, the build is passing.

/cc @famod (since you're the author of #14500) maybe you have some insights

**Expected behavior**
I can compile native with liquibase with views.

**Actual behavior**
mvn native compilation exit in error.

Command: 
```
mvn clean package -Pnative
```

Stack trace:
```
...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  33.586 s
[INFO] Finished at: 2021-02-01T12:48:24+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:1.11.1.Final:build (default) on project sdm-schema: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
[ERROR] 	[error]: Build step io.quarkus.liquibase.deployment.LiquibaseProcessor#nativeImageConfiguration threw an exception: java.lang.NullPointerException
[ERROR] 	at java.base/java.util.Objects.requireNonNull(Objects.java:221)
[ERROR] 	at java.base/java.util.Optional.<init>(Optional.java:107)
[ERROR] 	at java.base/java.util.Optional.of(Optional.java:120)
[ERROR] 	at io.quarkus.liquibase.deployment.LiquibaseProcessor.extractChangeFile(LiquibaseProcessor.java:345)
[ERROR] 	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
[ERROR] 	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655)
[ERROR] 	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
[ERROR] 	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
[ERROR] 	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
[ERROR] 	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
[ERROR] 	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
[ERROR] 	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
[ERROR] 	at io.quarkus.liquibase.deployment.LiquibaseProcessor.findAllChangeLogFiles(LiquibaseProcessor.java:316)
[ERROR] 	at io.quarkus.liquibase.deployment.LiquibaseProcessor.getChangeLogs(LiquibaseProcessor.java:276)
[ERROR] 	at io.quarkus.liquibase.deployment.LiquibaseProcessor.nativeImageConfiguration(LiquibaseProcessor.java:120)
[ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[ERROR] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[ERROR] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[ERROR] 	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:920)
[ERROR] 	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2415)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1452)
[ERROR] 	at java.base/java.lang.Thread.run(Thread.java:834)
[ERROR] 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

**To Reproduce**

Have a liquibase change with a view, example:
```xml
    <changeSet id="view_last_schema" author="gc">
        <createView viewName="v_last_schema">
            SELECT s.*
            FROM (
                SELECT data_offer_state_uuid, schema_encoding, schema_source, MAX(created) as created
                FROM t_schema
                GROUP by data_offer_state_uuid, schema_encoding, schema_source
                ) as o
            INNER JOIN t_schema s USING(schema_encoding, schema_source, created);
        </createView>
    </changeSet>
```

**Configuration**
```yml
quarkus:
  liquibase:
    migrate-at-start: true
```

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`:  `Linux gaetan-desktop 5.4.0-64-generic #72-Ubuntu SMP Fri Jan 15 10:27:54 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux`
 - Output of `java -version`:  
 ```
 openjdk version "11.0.10" 2021-01-19
OpenJDK Runtime Environment GraalVM CE 21.0.0 (build 11.0.10+8-jvmci-21.0-b06)
OpenJDK 64-Bit Server VM GraalVM CE 21.0.0 (build 11.0.10+8-jvmci-21.0-b06, mixed mode, sharing)
```
 - Quarkus version or git rev: `1.11.1.Final`
 - Build tool (ie. output of `mvnw --version` or `gradlew --version`): 
 ```
 Apache Maven 3.6.3
Maven home: /usr/share/maven
Java version: 11.0.10, vendor: GraalVM Community, runtime: /home/gaetan/.sdkman/candidates/java/21.0.0.r11-grl
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "5.4.0-64-generic", arch: "amd64", family: "unix"
```

--- a/extensions/liquibase/deployment/src/main/java/io/quarkus/liquibase/deployment/LiquibaseProcessor.java
+++ b/extensions/liquibase/deployment/src/main/java/io/quarkus/liquibase/deployment/LiquibaseProcessor.java
@@ -333,16 +333,16 @@ class LiquibaseProcessor {
 
     private Optional<String> extractChangeFile(Change change) {
         if (change instanceof LoadDataChange) {
-            return Optional.of(((LoadDataChange) change).getFile());
+            return Optional.ofNullable(((LoadDataChange) change).getFile());
         }
         if (change instanceof SQLFileChange) {
-            return Optional.of(((SQLFileChange) change).getPath());
+            return Optional.ofNullable(((SQLFileChange) change).getPath());
         }
         if (change instanceof CreateProcedureChange) {
-            return Optional.of(((CreateProcedureChange) change).getPath());
+            return Optional.ofNullable(((CreateProcedureChange) change).getPath());
         }
         if (change instanceof CreateViewChange) {
-            return Optional.of(((CreateViewChange) change).getPath());
+            return Optional.ofNullable(((CreateViewChange) change).getPath());
         }
         return Optional.empty();
     }

--- a/extensions/liquibase/deployment/src/test/java/io/quarkus/liquibase/test/LiquibaseExtensionConfigDefaultDataSourceTest.java
+++ b/extensions/liquibase/deployment/src/test/java/io/quarkus/liquibase/test/LiquibaseExtensionConfigDefaultDataSourceTest.java
@@ -27,6 +27,7 @@ public class LiquibaseExtensionConfigDefaultDataSourceTest {
                     .addClass(LiquibaseExtensionConfigFixture.class)
                     .addAsResource("db/xml/changeLog.xml")
                     .addAsResource("db/xml/create-tables.xml")
+                    .addAsResource("db/xml/create-views.xml")
                     .addAsResource("db/xml/test/test.xml")
                     .addAsResource("config-for-default-datasource.properties", "application.properties"));
 

--- a/extensions/liquibase/deployment/src/test/java/io/quarkus/liquibase/test/LiquibaseExtensionConfigMultiDataSourcesTest.java
+++ b/extensions/liquibase/deployment/src/test/java/io/quarkus/liquibase/test/LiquibaseExtensionConfigMultiDataSourcesTest.java
@@ -46,6 +46,7 @@ public class LiquibaseExtensionConfigMultiDataSourcesTest {
                     .addAsResource("db/users/changeLog.xml")
                     .addAsResource("db/xml/changeLog.xml")
                     .addAsResource("db/xml/create-tables.xml")
+                    .addAsResource("db/xml/create-views.xml")
                     .addAsResource("db/xml/test/test.xml")
                     .addAsResource("config-for-multiple-datasources.properties", "application.properties"));
 

--- a/extensions/liquibase/deployment/src/test/java/io/quarkus/liquibase/test/LiquibaseExtensionConfigNamedDataSourceWithoutDefaultTest.java
+++ b/extensions/liquibase/deployment/src/test/java/io/quarkus/liquibase/test/LiquibaseExtensionConfigNamedDataSourceWithoutDefaultTest.java
@@ -32,6 +32,7 @@ public class LiquibaseExtensionConfigNamedDataSourceWithoutDefaultTest {
                     .addClass(LiquibaseExtensionConfigFixture.class)
                     .addAsResource("db/xml/changeLog.xml")
                     .addAsResource("db/xml/create-tables.xml")
+                    .addAsResource("db/xml/create-views.xml")
                     .addAsResource("db/xml/test/test.xml")
                     .addAsResource("config-for-named-datasource-without-default.properties", "application.properties"));
 

--- a/extensions/liquibase/deployment/src/test/java/io/quarkus/liquibase/test/LiquibaseExtensionLoadChangeLogTest.java
+++ b/extensions/liquibase/deployment/src/test/java/io/quarkus/liquibase/test/LiquibaseExtensionLoadChangeLogTest.java
@@ -25,6 +25,7 @@ public class LiquibaseExtensionLoadChangeLogTest {
             .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
                     .addAsResource("db/xml/changeLog.xml")
                     .addAsResource("db/xml/create-tables.xml")
+                    .addAsResource("db/xml/create-views.xml")
                     .addAsResource("db/xml/test/test.xml")
                     .addAsResource("load-change-log-config.properties", "application.properties"));
 
@@ -36,7 +37,8 @@ public class LiquibaseExtensionLoadChangeLogTest {
             assertEquals("db/xml/changeLog.xml", changelog.getFilePath());
             assertNotNull(changelog.getChangeSets());
             assertEquals("db/xml/create-tables.xml", changelog.getChangeSets().get(0).getFilePath());
-            assertEquals("db/xml/test/test.xml", changelog.getChangeSets().get(1).getFilePath());
+            assertEquals("db/xml/create-views.xml", changelog.getChangeSets().get(1).getFilePath());
+            assertEquals("db/xml/test/test.xml", changelog.getChangeSets().get(2).getFilePath());
         }
     }
 

--- a/extensions/liquibase/deployment/src/test/java/io/quarkus/liquibase/test/LiquibaseExtensionMigrateAtStartXmlChangeLogTest.java
+++ b/extensions/liquibase/deployment/src/test/java/io/quarkus/liquibase/test/LiquibaseExtensionMigrateAtStartXmlChangeLogTest.java
@@ -27,6 +27,7 @@ public class LiquibaseExtensionMigrateAtStartXmlChangeLogTest {
             .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
                     .addAsResource("db/xml/changeLog.xml")
                     .addAsResource("db/xml/create-tables.xml")
+                    .addAsResource("db/xml/create-views.xml")
                     .addAsResource("db/xml/test/test.xml")
                     .addAsResource("migrate-at-start-xml-config.properties", "application.properties"));
 
@@ -37,9 +38,10 @@ public class LiquibaseExtensionMigrateAtStartXmlChangeLogTest {
             List<ChangeSetStatus> status = liquibase.getChangeSetStatuses(liquibaseFactory.createContexts(),
                     liquibaseFactory.createLabels());
             assertNotNull(status);
-            assertEquals(2, status.size());
+            assertEquals(3, status.size());
             assertFalse(status.get(0).getWillRun());
             assertFalse(status.get(1).getWillRun());
+            assertFalse(status.get(2).getWillRun());
         }
     }
 }

--- a/extensions/liquibase/deployment/src/test/resources/db/xml/changeLog.xml
+++ b/extensions/liquibase/deployment/src/test/resources/db/xml/changeLog.xml
@@ -6,5 +6,7 @@
 
     <include relativeToChangelogFile="true" file="create-tables.xml" />
 
+    <include relativeToChangelogFile="true" file="create-views.xml" />
+
     <include relativeToChangelogFile="true" file="test/test.xml" />
 </databaseChangeLog>

--- a/extensions/liquibase/deployment/src/test/resources/db/xml/create-views.xml
+++ b/extensions/liquibase/deployment/src/test/resources/db/xml/create-views.xml
@@ -1,0 +1,14 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="dev (generated)" id="create-view-1">
+        <createView viewName="LIQUIBASE_VIEW">
+            SELECT DISTINCT NAME
+            FROM LIQUIBASE;
+        </createView>
+    </changeSet>
+
+</databaseChangeLog>

